app/jobs/sms_sender_confirmation_job.rb

```diff
@@ -4,7 +4,7 @@ class SmsSenderConfirmationJob < ActiveJob::Base
   def perform(code, mobile)
     TwilioService.new.send_sms(
       to: mobile,
-      body: "Your #{APP_NAME} phone confirmation code is: #{code}."
+      body: "#{code} is your #{APP_NAME} one-time passcode."
```

Comment:
My idea was to distinguish between confirmation codes, and login codes.  

I like what paypal does which is to start the message with `PayPal:`.

They currently have `PayPay: Your confirmation code is 1233565`.

I actually like that but i would settle for:  `PayPay: 123445 is your confirmation code`